api/v1/kataconfig_types.go

```diff
@@ -31,7 +31,7 @@ type KataConfigSpec struct {
 
 	// CheckNodeEligibility is used to detect the node(s) eligibility to run Kata containers.
 	// This is currently done through the use of the Node Feature Discovery Operator (NFD).
-	// For more information on how the check works, please refer to the sandboxed containers documentation - https://docs.openshift.com/container-platform/latest/sandboxed_containers/deploying-sandboxed-container-workloads.html
+	// For more information on how the check works, please refer to the sandboxed containers documentation - https://docs.redhat.com/en/documentation/openshift_sandboxed_containers/1.6/html-single/user_guide/index#about-node-eligibility-checks_about-osc
```

Comment:
assuming 1.6 is intentional here

Reply:
Unfortunately, that is a publishing platform limitation. Currently, version numbers are hard-coded in URLs. Maybe the new tooling platform will allow more flexibility.